weave/trace_server/interface/base_object_classes/annotation_column.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional
+
+import jsonschema
+from pydantic import Field, field_validator
+
+from weave.trace_server.interface.base_object_classes import base_object_def
+
+
+class AnnotationColumn(base_object_def.BaseObject):
```

Comment:
I keep going back and forth but I think this should actually be `AnnotationSpec`. Column doesn't really make that much sense.